src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public void setFirstSide(double firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public void setSecondSide(double secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    public double getFirstSide() {
+        return firstSide;
+    }
+
+    public double getSecondSide() {
+        return secondSide;
+    }
+
+    @Override
+    public double area() {
+        return firstSide * secondSide;
+    }
```

Comment:
The method name 'area' is a verb, which is not recommended for method names. Consider renaming it to something like 'calculateArea' to make it clear that it performs an action.